src/Utils.vala

```diff
@@ -129,15 +129,14 @@ namespace Terminal.Utils {
      *
      * @return true if safe, false if unsafe.
      */
-    public bool is_safe_paste (string text, out string msg) {
+    public bool is_safe_paste (string text, out string[]? msg_array) {
```

Comment:
Does this need to be nullable? I don't think this ever goes to a null check. Would returning an empty array be fine?

Reply:
Just avoids unnecessarily initializing an array which is not used but an empty array could be used if you prefer.